PokerGame/UnitTestProject1/HighCardHandTests.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using System.Collections.Generic;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using PokerEngine;
+
+namespace UnitTestProject1
```

Comment:
Rename the project in the Solution Explorer, it should rename all references.  It may not rename the folder the project resides in though, so there may be a bit of annoying hand-editing .csproj/.sln files to reflect the path change after a folder rename.

Reply:
i had to rename it in the sln but done commiting soonm :)